server/channels/app/slashcommands/command_exportlink.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package slashcommands
+
+import (
+	"path"
+	"strings"
+	"time"
+
+	"github.com/mattermost/logr/v2"
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/public/shared/i18n"
+	"github.com/mattermost/mattermost/server/v8/channels/app"
+	"github.com/mattermost/mattermost/server/v8/channels/app/request"
+	"github.com/mattermost/mattermost/server/v8/platform/shared/filestore"
+)
+
+type ExportLinkProvider struct {
+}
+
+const (
+	CmdExportLink = "exportlink"
+)
+
+func init() {
+	app.RegisterCommandProvider(&ExportLinkProvider{})
+}
+
+func (*ExportLinkProvider) GetTrigger() string {
+	return CmdExportLink
+}
+
+func (*ExportLinkProvider) GetCommand(a *app.App, T i18n.TranslateFunc) *model.Command {
+	if !a.Config().FeatureFlags.EnableExportDirectDownload {
+		return nil
+	}
+
+	if !*a.Config().FileSettings.DedicatedExportStore {
+		return nil
+	}
+
+	b := a.ExportFileBackend()
+	_, ok := b.(filestore.FileBackendWithLinkGenerator)
+	if !ok {
+		return nil
+	}
+
+	return &model.Command{
+		Trigger:          CmdExportLink,
+		AutoComplete:     true,
+		AutoCompleteDesc: T("api.command_exportlink.desc"),
+		AutoCompleteHint: T("api.command_exportlink.hint"),
+		DisplayName:      T("api.command_exportlink.name"),
+	}
+}
+
+func (*ExportLinkProvider) DoCommand(a *app.App, c request.CTX, args *model.CommandArgs, message string) *model.CommandResponse {
+	if !a.SessionHasPermissionTo(*c.Session(), model.PermissionManageSystem) {
+		return &model.CommandResponse{ResponseType: model.CommandResponseTypeEphemeral, Text: args.T("api.command_exportlink.permission.app_error")}
+	}
+
+	b := a.ExportFileBackend()
+	_, ok := b.(filestore.FileBackendWithLinkGenerator)
+	if !ok {
+		return &model.CommandResponse{ResponseType: model.CommandResponseTypeEphemeral, Text: args.T("api.command_exportlink.driver.app_error")}
+	}
+
+	file := ""
+	if message == "latest" {
```

Comment:
could you elaborate a bit more here? is this the export latest?

Reply:
Yeah. If the admin uses `/exportlink latest`, we'll try to find the latest generated export and return a link to this export

Reply:
wondering is it better to make this a `const`/enum  instead versus a text in the code?